ocaml/utils/config.mlp

```diff
@@ -110,6 +110,7 @@ and ast_intf_magic_number = "Caml1999N029"
 and cmxs_magic_number = "Caml1999D029"
 and cmt_magic_number = "Caml1999T029"
 and linear_magic_number = "Caml1999L029"
+and cfg_magic_number = "Caml1999G029"
```

Comment:
Yes, it's a very good point! Magic number for cmx in flambda-backend is different from upstream because the format is different, but cfg format is new so there won't be any clashes. However we should change the one for Linear. I'll make a separate PR to change them both.

Reply:
I see the CFG format is shared between backends,
I was more worried about possible slightly different
assumptions that would lead to difficulties if mixing
closure- and flambda-produced CFGs.

Reply:
Ah, I see! Same applies to Linear.

Reply:
I changed cfg_magic_number here to use 2021, and the others will be fixed in #275.